bundle/edu.gemini.qv.plugin/src/main/scala/edu/gemini/qv/plugin/table/ObservationTableModel.scala

```diff
@@ -368,20 +369,34 @@ object ObservationTableModel {
   // ==== Wrapper case classes for some values which allow us to define custom renderers and sorting. ===
   case class RaValue(ra: Double) extends Comparable[RaValue] {
     def compareTo(other: RaValue): Int = ra - other.ra match {
-      case d if d < 0 => -1
-      case d if d == 0 => 0
-      case d if d > 0 => 1
```

Comment:
`math.signum(ra - other.ra).toInt`

Somewhere the `ra` `Double`s are normalized, or we know we're working with normalized values?  Not not for the patch release of course but we can switch this to `edu.gemini.spModel.core.RightAscension` in the future and get normalization, formatting, ordering and more.